safehttp/plugins/xsrf/xsrf.go

```diff
@@ -34,40 +35,64 @@ type UserIdentifier interface {
 }
 
 // Interceptor implements XSRF protection. It requires an application key and a
-// storage service. The appKey uniquely identifies each registered service and
+// storage service. The AppKey uniquely identifies each registered service and
 // should have high entropy. The storage service supports retrieving ID's of the
-// application's users. Both the appKey and user ID are used in the XSRF
-// token generation algorithm.
+// application's users. Both the AppKey and user ID are used in the XSRF
+// token generation and verification algorithm.
 type Interceptor struct {
 	AppKey     string
 	Identifier UserIdentifier
 }
```

Comment:
Maybe we should provide a default constructor for `Interceptor` which takes an `Identifier` and generates a safe `AppKey`

Reply:
Well, you could have multiple `Identifier`s for the same application I guess, but you might want the key to be unique across the application, IDK. @empijei, @kele WDYT?